### ISSUE TITLE
SWAT4LS paper

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-project#539]
Paper describing cool internal CEDAR plumbing:

Abstract due Sept 22, paper due October 6th.

http://swat4ls.blogspot.com/2017/06/swat4ls-2017-cfp-is-out.html

**Comment by marcosmro** [comment]
They have a "Long papers - Software papers" submission type that could be a good fit for that work:

"- Software papers – New applications and tool descriptions addressing a topic of interest to the SWAT4LS audience."

**Comment by martinjoconnor** [comment]
Yup - should be a good fit. This should be a plumbing paper (since we already have the ISWC paper describing the system).

We should discuss next week.

Some random initial ideas:

(1) How we used Neo4j, Mongo and microservice-based architecture to build semantic knowledge acquisition and publishing pipeline.
(2) How we built a semantic model built on standard technologies (JSON Schema, JSON-LD) to drive system
(3) Use of JSON-LD to store and publish linked data.
(4) FAIR malarkey (baked in really, but important)
(5) Resource management (Workspace, permissions, groups etc.)

Should not be a problem but need to make sure we differentiate from EKAW, AMIA, ICBO and ISWC papers.
